app/mailers/dependents_application_failure_mailer.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+class DependentsApplicationFailureMailer < ApplicationMailer
+  def build(user_hash)
+    @user_hash = user_hash
```

Comment:
Does this instance variable assignment need to exist? Seems like the plain old input param `user_hash` would suffice.